### ISSUE TITLE
Duplicate error when no active machine

### ISSUE THREAD

**Issue by ehazlett** [docker/machine#379]
When there is no active machine, the  error is printed as many times as the number of machines.  For example:

```
ERRO[0000] There's a problem finding the active host
ERRO[0000] There's a problem finding the active host
ERRO[0000] There's a problem finding the active host
NAME          ACTIVE   DRIVER         STATE     URL
dev                    vmwarefusion   Running   tcp://172.16.157.147:2376
dev-do                 digitalocean   Running   tcp://1.2.3.4:2376
test-fusion            vmwarefusion   Running   tcp://172.16.157.134:2376
```

**Comment by sthulb** [comment]
That was mentioned in #262

**Comment by ehazlett** [comment]
Ah right.  Back to that :)

**Comment by ehazlett** [comment]
Closing as this was fixed in #591.
